gradle-plugin/src/test/groovy/io/realm/gradle/PluginTest.groovy

```diff
@@ -142,7 +142,7 @@ class PluginTest {
         assertEquals(2, project.buildscript.repositories.size())
         try {
             assertEquals(4, project.repositories.size()) // The Android plugin adds 3 different local repos
-        } catch (Exception e) {
+        } catch (Error e) {
```

Comment:
Maybe at this point, it should just be `Throwable`

Reply:
Yup, it should. But the code has been removed now. I found a better way to debug it :)